internal/sdkv2provider/schema_cloudflare_access_identity_provider.go

```diff
@@ -38,6 +38,7 @@ func resourceCloudflareAccessIdentityProviderSchema() map[string]*schema.Schema
 		"config": {
 			Type:        schema.TypeList,
 			Optional:    true,
+			Computed:    true,
```

Comment:
The config object might be generated by the API with some default fields.
(Same for scim-config)